Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM node:8-alpine
 
-EXPOSE 3000
-
 ARG NODE_ENV
 ENV NODE_ENV $NODE_ENV
 
-RUN mkdir /app
-WORKDIR /app
-ADD package.json yarn.lock /app/
-RUN yarn --pure-lockfile
-ADD . /app
+WORKDIR /usr/src/app
+COPY package*.json .
+RUN yarn install
```

Comment:
We are using yarn here, so we should add `yarn.lock` file and install with `--frozen-lockfile` to have [reproducible dependencies](https://yarnpkg.com/lang/en/docs/cli/install/).